src/sdk/model/edge/edge-gateway-settings/edge-gateway-ssh-settings.ts

```diff
@@ -0,0 +1,56 @@
+import { EdgeGatewaySSHSettingsJson } from './__json/edge-gateway-ssh-settings-json';
+
+/* istanbul ignore next: autogenerated */
+export class EdgeGatewaySSHSettings {
+
+  constructor(private _json: EdgeGatewaySSHSettingsJson) {
+  }
+
+  /**
+   * Get remote access.
+   * @returns {boolean}
+   */
+  get remoteAccess(): boolean {
+    return this._json.remote_access;
+  }
+
+  /**
+   * Get user name.
+   * @returns {string}
+   */
+  get userName(): string {
+    return this._json.user_name;
+  }
+
+  /**
+   * Get ssh login banner.
+   * @returns {string}
+   */
+  get sshLoginBanner(): string {
+    return this._json.ssh_login_banner;
+  }
+
+  /**
+   * Get password expiry.
+   * @returns {number}
+   */
+  get passwordExpiry(): number {
```

Comment:
Shouldn't it be a date ?

Reply:
No it's a number of days.

Reply:
Ho ok perfect. I just wanted to be sure. All good to me then.